setup.py

```diff
@@ -44,7 +44,6 @@
         "sphinxcontrib-bibtex",
         "myst-parser",
     ],
-    "entr": ["pyhive"],
```

Comment:
Looks like nothing is in here to include the ENTR package, is that intentional?

Reply:
I added a try/catch to the entr import as requested.